enclave/core/optee/gp.c

```diff
@@ -384,6 +384,10 @@ void oe_abort(void)
 {
     /* No return */
     TEE_Panic(TEE_ERROR_GENERIC);
+
+    /* Ensure that it is unreachable on ARM machines */
+    while (1)
+        ;
```

Comment:
nit: Put the semicolon on the same line as the while.

Reply:
I tried but git would tell me to run format-code, which then placed the semicolon in another line.

Reply:
oh, ok. No problem